backstop/scenarios/date-picker.js

```diff
@@ -14,6 +14,9 @@ module.exports = [
   scenario('Type Date & Time 24 hour format', {
     selectors: ['#demo-date-and-time-24h'],
   }),
+  scenario('Type Date Range', {
+    selectors: ['#demo-date-range'],
+  }),
 
   // Hover states
```

Comment:
Maybe add test for hovered and focused range day.

Reply:
Added tests, but hover does not work :/

Reply:
Is this still broken? Looks good to me

Reply:
Yeah, if you check image, there is no hover style in it :/

Reply:
I removed hover test.. Couldnt figure why it was not working..